### DISCUSSION TITLE
Add a small version number in Windows portable binaries?

### DISCUSSION THREAD

**Discussion by ddarfantasy** [ImageMagick/ImageMagick#5749] [General] [answered]
I notice that there's no small version number in Windows portable binaries (such as [ImageMagick-7.1.0-portable-Q16-HDRI-x64.zip](https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-portable-Q16-HDRI-x64.zip), there's no "-52" in the file name), which is causing inconvenience for package managers like scoop which requires a hash check before extracting its content. I think a small version number in the file name indicating a version update would be better?

**Comment by urban-warrior** [comment]
It would be simple enough to add a minor # to the portable binaries.  Final decision, however, is in the hands of @dlemstra.

**Comment by dlemstra** [comment, marked as answer]
Just pushed a patch to make sure this is changed in the next release.

**Reply by ddarfantasy** [reply]
I've noticed that there's a valid link at [ImageMagick-7.1.0-53-portable-Q16-HDRI-x64.zip](https://imagemagick.org/archive/binaries/ImageMagick-7.1.0-53-portable-Q16-HDRI-x64.zip) but the links in the official website is still old?

**Reply by urban-warrior** [reply]
Try now.

**Reply by ddarfantasy** [reply]
That's good, thank you all!

**Comment by ddarfantasy** [comment]
Sorry to bother, but could you remove the corresponding old zips from https://imagemagick.org/archive/binaries ? @urban-warrior

**Reply by ddarfantasy** [reply]
They are removed, thanks!
